gstools/covmodel/tools.py

```diff
@@ -576,11 +576,19 @@ def model_repr(model):  # pragma: no cover
     model : :any:`CovModel`
         The covariance model in use.
     """
-    opt_str = ""
+    if not np.isclose(model.rescale, model.default_rescale()):
+        opt_str = ", rescale={0:.{p}}".format(model.rescale, p=model._prec)
+    else:
+        opt_str = ""
     for opt in model.opt_arg:
-        opt_str += (
-            ", " + opt + "={0:.{1}}".format(getattr(model, opt), model._prec)
+        opt_str += ", {0}={1:.{p}}".format(
+            opt, getattr(model, opt), p=model._prec
```

Comment:
I think
`opt_str += f", {opt}={getattr(model, opt):.{model._prec}}"`
looks much cleaner, f-strings WTF!

Reply:
You're right! f-strings FTW!